content/en/docs/collector/configuration.md

```diff
@@ -448,3 +448,94 @@ following proxy environment variables:
 
 If set at Collector start time then exporters, regardless of protocol, will or
 will not proxy traffic as defined by these environment variables.
+
+### Authentication
+
+Most receivers exposing an HTTP or gRPC port are able to be protected using the collector's
+authentication mechanism, and most exporters using HTTP or gRPC clients are able to add
+authentication data to the outgoing requests.
+
+The authentication mechanism in the collector uses the extensions mechanism, allowing for
+custom authenticators to be plugged into collector distributions. If you are interested
+in developing a custom authenticator, check out the "Building a custom authenticator" document.
+
+Each authentication extension has two possible usages: as client authenticator for exporters,
+adding auth data to outgoing requests, and as server authenticator for receivers, authenticating
+incoming connections. Refer to the authentication extension for a list of its capabilities, but
+in general, an authentication extension would only implement one of those traits. For a list of
+known authenticators, use the [Registry](/registry/?s=authenticator&component=extension&language=) available in this website.
+
+To add a server authenticator to a receiver in your collector, make sure to:
+
+1. add the authenticator extension under the `extensions` section
+1. add the extension to the `services` node
```

Comment:
nit - Having both words "section" and "node" for describing configuration is confusing. Do you believe dot notation would help with describing the area where configuration is needed?

Reply:
Done! PR updated.